ITE/5/README.adoc

```diff
@@ -0,0 +1,153 @@
+= ITE-5: Replace signature envelope with SSL signing spec
+:source-highlighter: pygments
+:toc: preamble
+:toclevels: 2
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 5
+
+| Title
+| Replace signature envelope with SSL signing spec
+
+| Sponsor
+| link:https://github.com/santiagotorres[Santiago Torres]
+
+| Status
+| Draft 💬
+
+
+| Type
+| Standards
+
+| Created
+| 2020-09-28
+
+|===
+
+[[abstract]]
+== Abstract
+
+This link:https://github.com/in-toto/ITE[in-toto enhancement (ITE)] proposes
+removing the in-toto signature scheme requirements, and allow users to adopt the
+signature envelope that better fits their ecosystem. This has the following
+general benefits:
+
+1. avoid mandating an ad-hoc signing specification for in-toto; and
+2. allow for implementers to propose different signing specifications.
+
+In addition, this ITE makes a recommendation to use the
+link:http://github.com/secure-systems-lab/signing-spec[SSL Signing Spec]. This
+has the following benefits over the current state:
+
+1. Avoids canonicalization for security reasons (i.e., to not parse untrusted input) 
+2. Reduces the possibility of misinterpretation of the payload. The serialized payload is encoded as a string and verified by the recipient before de-serializing.
+
+[[specification]]
+== Specification
+
+The specification adopted will be the SSL signing spec, as linked above. As
+such, we defer to that document to describe the specifics of signature
+generation and verification.
+
+The envelope's `payloadType` is either:
+
+* `https://in-toto.io/Link/v1+json` (for links)
```

Comment:
@SantiagoTorres What do you think about these URLs?

Reply:
Santiago mentioned possible adding "/schemas" to the start.

Another question: should we have just one URI for both types, since the `_type` field within the payload indicates the type?

Reply:
Done. There is now a single URI and it includes as "schemas/" path prefix. Please take a look.